### DISCUSSION TITLE
How can i find maxZoom Value with getMaxZoom method in CameraAwesomeBuilder.custom?

### DISCUSSION THREAD

**Discussion by dasaki-gr** [Apparence-io/CamerAwesome#227] [Q&A]
Congrats on the plugin. 
I would like some help. 
What is the most correct way to find the maxZoom value in the method onPreviewScaleBuilder with the use of getMaxZoom method?
I find that onPreviewScaleBuilder i can use  OnPreviewScale(onScale: ((scale) {..... but scale value is only between 0 and 1.

**Comment by apalala-dev** [comment]
Thanks!

The scale value in onScale is a percentage. 0 means it's the minimal zoom while 1 is the maximal zoom.
Depending on the phone capabilities, the zoom that one might want to display to an user will differ. 0 could be x1 for a classic phone, but if there's a wide angle, it could be x0.6 for example.
Max zoom could be x2, x4, or even x30 for a Pixel 7 Pro for instance.

You are trying to retrieve which zoom to display to the user ? (x1, x4, x30...)

**Reply by dasaki-gr** [reply]
Thanks for your reply.
Yes, I want to retrieve the actual zoom every time so that I can change some drawings that I display on the screen accordingly and they should be enlarged correctly. E.g. if the magnification is 3x I should enlarge the drawings by 3 times.
So I need to know the maximum zoom corresponding to scale 1 (e.g. 5x) and then calculate the correct zoom at each scale respectively (e.g. scale 0,5 => 2,5x)
So the question is how I will use the method getMaxZoom() inside the  CameraAwesomeBuilder.custom( builder: ... onPreviewScaleBuilder(...   )
